packages/app-store/routing-forms/lib/getAttributes.ts

```diff
@@ -113,38 +65,3 @@ type AttributeOptionValueWithType = {
 };
 
 type UserId = number;
-
-export async function getTeamMembersWithAttributeOptionValuePerAttribute({ teamId }: { teamId: number }) {
```

Comment:
This is now done in getSerializeForm- which already was actually already doing some teamMember logic but unused. getSerializeForm is also deprecating getAttributesFromTeam in the responseHandler - but remains in this file as it is used in the getAttributesForTeam (used in organisation settings)

Using teamMember attributes we are able to reduce the getAttributesForTeam